src/coreclr/debug/di/process.cpp

```diff
@@ -7719,6 +7721,7 @@ HRESULT CordbProcess::GetRuntimeOffsets()
             m_runtimeOffsets.m_signalHijackCompleteBPAddr,
             m_runtimeOffsets.m_excepNotForRuntimeBPAddr,
             m_runtimeOffsets.m_notifyRSOfSyncCompleteBPAddr,
+            m_runtimeOffsets.m_setThreadContextNeededAddr,
```

Comment:
Lets move the #ifdef FEATURE_INTEROP_DEBUGGING so it only covers the first six flares and get rid of the comment that says flares are only for interop debugging.